native/src/components/__tests__/ProgressSpinner.spec.tsx

```diff
@@ -10,7 +10,7 @@ type DeepPartial<T> = {
 describe('ProgressSpinner', () => {
   let theme: DeepPartial<ThemeType>
 
-  const t = input => input
+  const t = (input: string) => input
```

Comment:
Maybe we should provide this as export in a testing utils file, maybe always mapping the key to `t_${key}` like in your last PR in the future to be able to assert that keys are actually translated. What do you think? You don't need to do this here of course though.

Reply:
I think it is fine to redo this in every test. Then the code which is executed is not split across files. In tests I kind of like repetition.

Reply:
I also thought about that. But that would mean quite some refactoring for all tests, as all `getByText` calls would also need to change. And I am not sure whether that always makes sense. I will create an issue and we can then discuss this in our next grooming :)